docker-stack.yml

```diff
@@ -29,8 +35,14 @@ services:
         delay: 10s
         max_attempts: 3
         window: 120s
+      resources:
```

Comment:
Your elasticsearch service failed to start in the docker cluster - my guess is this is insufficient allocation for the container. [Guidelines](https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html) don't seem to specify a requirement, maybe we can look at usage stats for other stacks and see what they are.

Reply:
Trying 2 GB limits with 1 GB reservations. We'll see how the next deployment goes.